mantidproject/mantid
# 16540 hfir sans file new format

Description of work.

See Issue #16540 

To test:

Load one old and one new file. 
You can also open the GUI and see if the fields are properly populated.

Fixes #16540 .

No release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
